src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -37,7 +37,7 @@ def get_checksum_error_value(self, dmg, device_id):
             for device in devices:
                 try:
                     if device['uuid'] == device_id:
-                        return device['health']['checksum_errs']
+                        return device['ctrlr']['health_stats']['checksum_errs']
```

Comment:
Unfortunately, this test isn't covered by `Features: control pool`

You can use `Features: control pool CsumErrorLog`

Or, if you think it should be covered under `control` or `pool`, you can add that tag to the third line of tags here:
https://github.com/daos-stack/daos/blob/bcdc5487c7f9ce1a37fb92ab051a6a8e8c21f5dd/src/tests/ftest/checksum/csum_error_logging.py#L55

Reply:
Currently running with `Test-tag: vmd,vmd_led nvme,pool checksum,faults,daos_test pool,pool_create_all` which should cover most of the fixes.